com.parsecgaming.parsec.yml

```diff
@@ -0,0 +1,74 @@
+---
+app-id: "com.parsecgaming.parsec"
+runtime: "org.freedesktop.Platform"
+runtime-version: "19.08"
+sdk: "org.freedesktop.Sdk"
+command: "parsec"
+copy-icon: true
+separate-locales: false
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--device=dri"
+  - "--persist=."
```

Comment:
Can you remind me what this hack is for?

Reply:
This is required because parsec ignores the xdg-standard and stores all sorts of things in `$HOME` from the login state to the actual library that is executed as well as settings, ….

Original I symlinked the files to the `XDG_DATA_HOME` but when rewriting I decided that using the `--persist` flag is easier to achive the same thing.

Reply:
Is there a way to tell the app to use $XDG_DATA_HOME instead?

Reply:
I asked again in their support channel as documentation for parameter or alike is basically non-existent. No answer so far.

According to some community member, Linux support is rather "existent" than maintained or in line with any linux-based development standard. So I don't think that's going to happen.

I could switch to the old approach of symlinking the files they expect in `$HOME` to `$XDG_DATA_HOME` but this is more likely to break on any release.